ibis/backends/tests/test_client.py

```diff
@@ -70,17 +75,22 @@ def _create_temp_table_with_schema(con, temp_table_name, schema, data=None):
 @pytest.mark.parametrize(
     "sch",
     [
-        ibis.schema(
-            [
-                ("first_name", "string"),
-                ("last_name", "string"),
-                ("department_name", "string"),
-                ("salary", "float64"),
-            ]
+        param(
```

Comment:
Is there a reason for this diff?

Reply:
I think that appeared after I switched to using `param` for the parameters.